quiche/src/lib.rs

```diff
@@ -8306,7 +8426,26 @@ impl TransportParams {
             TransportParams::encode_param(&mut b, 0x000c, 0)?;
         }
 
-        // TODO: encode preferred_address
+        if is_server {
+            if let Some(preferred_address_params) = &tp.preferred_address_params {
+                /// IPv4: 4 bytes
+                /// Port: 2 bytes
+                /// IPv6: 16 bytes
```

Comment:
nit: these should use double quotes //

Reply:
I believe my original intention was to have these serve as "Documentation Comments" to the constant below. This is what it looks like when the documentation for the co
<img width="419" alt="Screenshot 2024-07-30 at 11 57 40 AM" src="https://github.com/user-attachments/assets/f38a4c4a-47a8-48f6-9d5b-c9ad94983c8a">
nstant is pulled up in my editor:

Reply:
Lucas: "Switch these to regular //"